packages/compass-e2e-tests/tests/data-modeling-tab.test.ts

```diff
@@ -834,5 +927,99 @@ describe('Data Modeling tab', function () {
       await browser.clickVisible(Selectors.DataModelUndoButton);
       await getDiagramNodes(browser, 2);
     });
+
+    it('allows field editing', async function () {
+      const dataModelName = 'Test Edit Collection';
+      await setupDiagram(browser, {
+        diagramName: dataModelName,
+        connectionName: DEFAULT_CONNECTION_NAME_1,
+        databaseName: 'test',
+      });
+
+      const dataModelEditor = browser.$(Selectors.DataModelEditor);
+      await dataModelEditor.waitForDisplayed();
+
+      let collectionText = await browser
+        .$(Selectors.DataModelPreviewCollection('test.testCollection-one'))
+        .getText();
+      expect(collectionText).to.not.include('field-1');
+
+      // Drag the node to show add new field buttons and new fields.
+      await dragNode(
+        browser,
+        Selectors.DataModelPreviewCollection('test.testCollection-one'),
+        { x: -100, y: -100 }
```

Comment:
Looks like some of the CI machine sizes are different from what's happening on my desktop for this test. 
```suggestion
        { x: -100, y: 0 }
```
https://github.com/mongodb-js/compass/blob/7f8d06c283268f28fbaa38d81178f1acb0ea1d21/packages/compass-e2e-tests/package.json#L14 It should be 1432x840, and the screenshots are showing that size, however these interactions are still happening differently on my desktop vs e2e.